CHANGELOG.MD

```diff
@@ -16,6 +16,8 @@
 * Scroll cursor in input to the end of the text when triggering autocomplete.
 * Add `advanced-open-file:delete-path-component` command for easily navigating
   directory hierarchies. (artagnon)
+* Add `advanced-open-file:confirm-selected-or-first` command for easier opening
```

Comment:
This should be under a new pending version (which I added a short while ago), I'll rebase and then move it.